.github/workflows/publish.yml

```diff
@@ -68,6 +77,50 @@ jobs:
           shopt -s globstar
           docker run -v "${PWD}:${PWD}" ghcr.io/linz/argo-tasks:v3 stac-validate "$PWD"/stac/**/collection.json
 
+      - name: Setup kubectl
+        if: ${{ !contains(github.event.head_commit.message, '[skip-argo]')  }}
```

Comment:
These `if` statements are necessary to make the job semantically equivalent to the original code, but we should probably remove them unless there's a good reason to skip these steps regularly. That said, it could be done in a separate PR.

Reply:
I agree, this functionality has never been used and I can't see a reason that it would be, especially now that the parameter files are automatically generated.

I'll remove it in a separate PR so that we have this in history just incase we want to add it back.